### DISCUSSION TITLE
Site.IsMultiLingual is deprecated

### DISCUSSION THREAD

**Discussion by cyber700** [McShelby/hugo-theme-relearn#910] [General]
On build I got the following:
Start building sites … 
hugo v0.134.1+extended darwin/amd64 BuildDate=2024-09-05T10:17:50Z VendorInfo=brew

WARN  deprecated: .Sites.First was deprecated in Hugo v0.127.0 and will be removed in a future release. Use .Sites.Default instead.
WARN  deprecated: .Site.IsMultiLingual was deprecated in Hugo v0.124.0 and will be removed in a future release. Use hugo.IsMultilingual instead.

**Comment by McShelby** [comment]
That's expected as the theme is still downward compatible to 0.121.0
